src/pages/BorrowerProfile/BorrowerProfile.vue

```diff
@@ -179,7 +185,7 @@ const preFetchQuery = gql`
 				key
 				value
 			}
-			userContext: uiExperimentSetting(key: "new_users_context") {
```

Comment:
MARS confirmed this experiment setting can be removed here. This is the only reference in the code.